README.md

```diff
@@ -13,26 +13,27 @@ const {
 } = utils;
```

Comment:
I noticed you changed this import flow in jitsi-meet-electron to use destructuring assignment for RemoteContorl and setupScreenSharingForWindow. Maybe you should do the same here for consistency?